src/main/data/facilities.json

```diff
@@ -4016,6 +4016,10 @@
   {
     "code": "TNR",
     "description": "THE NEW RESIDENCE"
```

Comment:
A comma is required after this line to resolve the merge issue

Reply:
Damn commas. Thanks, I was never a big fan of assembly language... :) I think I fixed it. But one never knows.